### ISSUE TITLE
Integrate Roku into the system and wright custom software for remote control and media player

### ISSUE THREAD

**Issue by omerll** [oceanbuilders/smartpod#31]
**Acceptance Criteria:**

1.- Integrate Roku into the system
2.- Wright code for Media player(play/pause, next and forward, volume control, channel selection, on/off)
3.- Wright code for Roku Remote(arrow up, arrow down, arrow left, arrow right, ok button, back button, home button, volume + and volume -)

**Comment by omerll** [comment]
UI made for the Roku Control
![photo_2021-06-28_13-14-52](https://user-images.githubusercontent.com/84348234/123684300-ebae1f00-d812-11eb-913f-bf00134a436b.jpg)

Demo on the Entrainment system. The screen will be replaced with the projector. Sound is amazing. Speech and music is Loud and clear (is Dolby audio certified, 4 speakers 🔊 inside the sounds bar) 😻 just missing the subwoofer that we will integrate. But I’m really amazed how much sound power  it makes for its size and power consumption. The Software/integration with SeapodHome automation system is done ☑️. Just waiting on the proyector screen and proyector to finish the hole system 💪🏻
https://user-images.githubusercontent.com/84348234/123684080-a38efc80-d812-11eb-9b76-a9eed2aaa7ef.mp4

Now we can throw away this remote =)
![roku](https://user-images.githubusercontent.com/84348234/123684690-637c4980-d813-11eb-9858-dc54f026e7c2.jpg)

**Comment by scottdear** [comment]
This needs to have a pull request with source code checked in to the smartpod repo, also README on how to build would be cool too.

**Comment by omerll** [comment]
In this file we have all the scripts that make the magic and comunicate with the roku device: https://github.com/oceanbuilders/Seapod-HA-V1/blob/main/scripts.yaml

Here is the UI, it connects the scripts that i wrote with buttons of the roku remote: the code starts in line 200 and ends in line 275 =) https://github.com/oceanbuilders/Seapod-HA-V1/blob/main/lovelaceui.yaml

**Comment by scottdear** [comment]
This stuff needs to be checked into either HASS integrations or HASS add-ons as a pull request.  Once the pull request ready, then these stories need to remove to review.  Also we need the source code for the scripts you mentioned checked in.  In short, all code needs to be checked into github, checking in the configuration files is a good start.  We also need to deprecate the Seapod-HA-V1 repository and check things into the smartpod, hass-addons repositories, let me know if you have any questions on this.

**Comment by omerll** [comment]
We will open a new issue for this to make the HASS integration with the help of Ola and Scott. 
Design process done
